src/Jab/ContainerGenerator.cs

```diff
@@ -208,7 +212,7 @@ private void Execute(GeneratorContext context)
 
                         foreach (var rootService in root.RootCallSites)
                         {
-                            var rootServiceType = rootService.ServiceType;
+                            string rootServiceType = rootService.ServiceTypeString;
```

Comment:
With CodeWriter change, do we need ServiceTypeString and ServiceType.ToDisplayString(NullableFlowState.NotNull) usages in this file?

Reply:
CodeWriter has special handling of interpolation: https://github.com/pakrym/jab/blob/main/src/Jab/CodeWriter.cs#L61  so all INamedTypeSymbol formatting parameters will go through AppendType

Reply:
Nice, I haven`t noticed this

Reply:
Fixed